package.json

```diff
@@ -10,7 +10,7 @@
     "console": "node --experimental-repl-await console.js",
     "cover": "npm run test -- --coverage",
     "test": "NODE_ENV=testing jest --runInBand --forceExit --detectOpenHandles --passWithNoTests",
-    "test-inspect": "NODE_ENV=testing node --inspect --debug-brk jest",
+    "test-inspect": "NODE_ENV=testing node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
I think it will work too with `jest` instead of `node_modules/.bin/jest`

Reply:
![image](https://user-images.githubusercontent.com/28768483/81424895-031cdb00-912d-11ea-8f04-c8f8b210ba02.png)
Doesn't works 😕